Facerecognition.py

```diff
@@ -0,0 +1,18 @@
+from distutils.log import error
+from logging import exception
+import face_recognition
+import os
+import Constants
+
+list_encoding = []
+list_all_encodings = []
+class Facerecognition:
+    def __init__(self) -> None:
+        try:
+            for images in os.listdir(Constants.IMAGE_PATH):
+                Image_loading = face_recognition.load_image_file(Constants.IMAGE_PATH + "/" + images)
```

Comment:
A separate method is required . Pass image path as an argument and let the method return encodings for each image